plugins/pom.xml

```diff
@@ -78,6 +78,60 @@
 
         <module>org.jkiss.dbeaver.headless</module>
     </modules>
+    <dependencies>
```

Comment:
Oh, now, please just add 
<module>org.jkiss.dbeaver.ext.databend</module>
higher

and 
<module>org.jkiss.dbeaver.ext.databend.ui</module>

lower

Reply:
And delete all these changes

Reply:
> Oh, now, please just add org.jkiss.dbeaver.ext.databend higher
> 
> and org.jkiss.dbeaver.ext.databend.ui
> 
> lower

Emm Would you like to explain what is the mean of `higher` or `lower`? 😅

Reply:
Somewhere here
https://github.com/dbeaver/dbeaver/blob/devel/plugins/pom.xml#L31
and here
https://github.com/dbeaver/dbeaver/blob/devel/plugins/pom.xml#L97

Like other plugins